docs/Features/XMLReferencesFeatures.md

```diff
@@ -49,9 +53,12 @@ After saving this setting, you will get completion, go to definition, and highli
 
 The `xml.references` settings is an array of objects with the following properties:
 
- * `pattern`:  matches the files that reference declared with `expressions` applies to. See [glob syntax](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob) for more information about the pattern syntax. 
+ * `pattern`:  matches the files that reference declared with `expressions` applies to. See [glob syntax](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob) for more information about the pattern syntax.
+ * `prefix (optional)`: the prefix to use (ex : '#')  for from for all expressions.
```

Comment:
I think it would be more clear if this said:

"...for all the declared reference expressions."